spec/models/event_spec.rb

```diff
@@ -77,10 +77,15 @@
     let(:event_001) { create :event, start_time: 2.days.from_now, end_time: 4.days.from_now }
     let(:event_002) { create :event, start_time: 3.days.from_now, end_time: 7.days.from_now }
     let(:event_003) { create :event, start_time: 7.days.from_now, end_time: 9.days.from_now }
-    before { setup }
+
+    before do
+      event_000
+      event_001
+      event_002
+      event_003
```

Comment:
I think you have these lines because oitherwise the `let` statements won't happen ( lazy evaluation ) ? You can accomplish that by using `let!` in line 79, then the object will always be created. If that does what you want, we could drop this entire block.

Reply:
ok